{
  "type": "class",
  "name": "BlockModel.LoopException",
  "qualified_name": "net.minecraft.client.renderer.block.model.BlockModel.LoopException",
  "package": "net.minecraft.client.renderer.block.model",
  "modifiers": [
    "public",
    "static",
    "class"
  ],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [
      "Serializable"
    ],
    "inheritance_tree": [
      "java.lang.Object",
      "java.lang.Throwable",
      "java.lang.Exception",
      "java.lang.RuntimeException"
    ]
  },
  "nested_classes": [],
  "fields": [],
  "constructors": [
    {
      "name": "LoopException",
      "modifiers": [
        "public"
      ],
      "annotations": [],
      "parameters": [
        {
          "name": "p_173424_",
          "type": "String",
          "annotations": []
        }
      ],
      "exceptions": [],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "1.21.x-neoforge/net/minecraft/client/renderer/block/model/BlockModel.LoopException.html"
}